infra/versions.tf

```
@@ -19,12 +19,12 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 4.84.1"
+      version = ">= 3.53, < 6"
     }
 
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = ">= 3.53, < 5.0"
+      version = ">= 3.53, < 6"
```

nit: generally best to match the TPG versions

```suggestion
      version = ">= 4.83.0, < 6"
```